src/Filter/ScssFilter.php

```diff
@@ -54,7 +54,11 @@ public function input($filename, $input)
             return $input;
         }
         $filename = preg_replace('/ /', '\\ ', $filename);
-        $bin = $this->_settings['sass'] . ' ' . $filename;
+        $cmd = $this->_settings['sass'];
+        foreach ($this->_settings['imports'] as $path) {
+            $cmd .= " -I \"{$path}\"";
```

Comment:
Should these segments have `"` escaped before being added into the shell snippet?